source/common/upstream/cds_api_impl.h

```diff
@@ -53,10 +53,10 @@ class CdsApiImpl : public CdsApi,
   void runInitializeCallbackIfAny();
 
   ClusterManager& cm_;
+  Stats::ScopePtr scope_;
```

Comment:
Would be good to have a test to detect if member order ever regresses.

Reply:
Yes I agree. I think this should be possible with mocks per above. cc @krnowak if you could follow up and add this.